### PR TITLE
adds carks donor item. Jamie said no more funny titles so this will be the very last one because big dad word is big dad law.

### PR DESCRIPTION
**Pull request by KillerOrcaCora**
Adds Donor item for cark 

# Spriting
image
pride1
pride2
pride3
pride4
pride5
pride6

If you want a donor item done by me DM me but my prices are super high and I don't work with poor people.
# Changelog

:cl:  

imageadd: Extending your pride month cost money now.

/:cl:

### CODE REVIEW

**Comment by MajManatee** [comment]
feature freeze. no prior approval

**Comment by KillerOrcaCora** [comment]
was approved

**Comment by ThatLing** [comment]
Conflicts with preferences rework

**Comment by MajManatee** [comment]
why did it take you 30 comits to add like 2 sprites

File: yogstation/code/modules/clothing/donator.dm [hunk 1]
```diff
@@ -184,3 +184,22 @@
...
+	item_state = "sheetpride"
+	slot_flags = ITEM_SLOT_NECK
+	layer = MOB_LAYER
+	throwforce = 0
+	throw_speed = 1
+	throw_range = 2
+	w_class = WEIGHT_CLASS_TINY
+	resistance_flags = FLAMMABLE
+	dying_key = DYE_REGISTRY_BEDSHEET
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF // To prevent our lessers from doing very cringe things
```
Comment (JamieD1) [inline yogstation/code/modules/clothing/donator.dm]:
```suggestion
```

**Review by JamieD1 — APPROVED** [review]

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**